src/main/java/propertyFiles/Bundle.properties

```diff
@@ -2383,6 +2383,7 @@ staticSearchFields.dvObjectType=Type
 staticSearchFields.fileTag=File Tag
 staticSearchFields.fileAccess=Access
 staticSearchFields.publicationStatus=Publication Status
+staticSearchFields.subject_ss=Subject
```

Comment:
@JayanthyChengan is this supposed to be part of this PR?

Reply:
@scolapasta  - Missing term related to facet Internationalization.  hope it's okay to add with this PR.